src/redux/reducers/liquorSearchReducer.js

```diff
@@ -0,0 +1,27 @@
+
+export const initialState = {
+    allDrinksData: [],
+    liquorSearchWord: '',
+    randomDrink: {}
+}
+
+export const liquorSearchReducer = (state = initialState, action) => {
+    switch(action.type) {
+        case 'LOAD_DRINK_DATA':
+            return {
+                allDrinksData: [...state.allDrinksData, action.payload]
```

Comment:
same question as in boardGameReducer - is the spread operator required here to destructure state on line 11.5 ?

Reply:
I think we should 👍🏻